test/feature/step_definitions/setup_definitions.coffee

```diff
@@ -1,17 +1,26 @@
 { expect } = require 'chai'
+coffee = require 'coffee-script'
 
 
 module.exports = ->
 
   @Given /^an exprestive app using defaults$/, (done) ->
-    @createExprestiveApp '', (err) =>
+    @createExprestiveApp library: 'connect', (err) =>
+      return done err if err
```

Comment:
I'm going to ship this PR for now, I'll accept a future PR implementing nitroglycerine if you can get @charlierudolph to agree to it. Otherwise let's refactor these tests to use promises in a future PR